plugins/utilities/mood_light.py

```diff
@@ -0,0 +1,253 @@
+# ba_meta require api 7
+from __future__ import annotations
+from typing import TYPE_CHECKING, cast
+
+import ba
+import _ba
+import random
+from ba._map import Map
+from bastd import mainmenu
+from bastd.ui.party import PartyWindow
+from bastd.gameutils import SharedObjects
+from time import sleep
+if TYPE_CHECKING:
+    from typing import Any, Sequence, Callable, List, Dict, Tuple, Optional, Union
+
+# mood light plugin by ʟօʊքɢǟʀօʊ
+
+
+def Print(arg1, arg2="", arg3=""):
+    ba.screenmessage(str(arg1)+str(arg2)+str(arg3))
+
+
+try:
+    with open("moodlightSettings.txt", "r") as mltxt:
+        global Ldefault, Udefault
+        data = mltxt.read()
+        Ldefault, Udefault = data.split("\n")
+        Ldefault = int(Ldefault)
+        Udefault = int(Udefault)
+except:
+    with open("moodlightSettings.txt", "w") as mltxt:
+        mltxt.write("15 \n 20")
```

Comment:
let's avoid creating a new file for this, can we store these values in config.json?

Reply:
Yeah i planned on doing that in the next update

Reply:
oh i see, let's do it now since it'll keep zombie files around after the update

Reply:
Yeah ok